utils/diagram_generator/lib/main.dart

```diff
@@ -62,6 +62,8 @@ Future<void> main() async {
   final List<String> categories = flags['category'] as List<String>;
   final List<String> names = flags['name'] as List<String>;
 
+  print('Filters:\n  categories: $categories\n  names: $names');
```

Comment:
Nit: Was this meant to be for debug or actually included here?

Reply:
It was intended to stay. The generator is really only run by us, so it's useful to see what it's doing. Previously we didn't report what filters we applied. (This is especially helpful on Linux where we don't currently handle arguments so the filters you specify on the command line don't actually apply and previously that was very confusing.)